app/src/main/java/org/schabi/newpipe/info_list/dialog/InfoItemDialog.java

```diff
@@ -330,6 +330,7 @@ public Builder addDefaultEndEntries() {
             addPlayWithKodiEntryIfNeeded();
             addMarkAsWatchedEntryIfNeeded();
             addEntry(StreamDialogDefaultEntry.SHOW_CHANNEL_DETAILS);
+            addEntry(StreamDialogDefaultEntry.SHOW_TOPIC);
```

Comment:
This should not be put here. It is a really really rare action and should be put as a section in the description tab in `VideoDetailsFragment`